recipes/rapidxml/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
+            self.run(bin_path, run_environment=True)
```

Comment:
And last but not least, this would become:
```
            xml_path = os.path.join(self.source_path, "beer_journal.xml")
            self.run("{} {}".format(bin_path, xml_path), run_environment=True)
```